apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/api/Sort.java

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Copyright © 2018-2021 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.apl.util.api;
+
+/**
+ * @author Andrii Boiarskyi
+ * @see
+ * @since 1.0.0
```

Comment:
Probably version better to update to the correct: 1.50.32 or 1.48.11

Reply:
1.50.31;